api/src/queries/survey/survey-occurrence-queries.ts

```diff
@@ -197,14 +197,13 @@ export const deleteSurveyOccurrencesSQL = (occurrenceSubmissionId: number): SQLS
 /**
  * SQL query to get the record for a single occurrence.
  *
- * @param {number} surveyId
  * @param {number} submissionId
  * @returns {SQLStatement} sql query object
  */
-export const getSurveySubmissionOccurrenceSQL = (surveyId: number, submissionId: number): SQLStatement | null => {
-  defaultLog.debug({ label: 'getSurveySubmissionOccurrenceSQL', message: 'params', surveyId });
+export const getSurveySubmissionOccurrenceSQL = (submissionId: number): SQLStatement | null => {
+  defaultLog.debug({ label: 'getSurveySubmissionOccurrenceSQL', message: 'params', submissionId });
 
-  if (!surveyId || !submissionId) {
```

Comment:
same here  .. when I originally coded these up, I had the surveyId, but I got feedback to remove them because they are not stricly used in query.